### PR TITLE
Don't prevent rendering in browser when scroll at 0

### PR DESCRIPTION
**Pull request by hssm**
@timrae Do you remember exactly why this condition was included? I couldn't understand why it was there and there appear to be no adverse effects of removing it as far as I can tell. It fixes https://github.com/ankidroid/Anki-Android/issues/4339.

### CODE REVIEW

**Comment by timrae** [comment]
I do remember struggling with some issue with the async task and scrolling here, but I introduced that condition in a previous iteration of the code (which worked quite differently), and I don't remember the specifics of what the issue was, or whether it's still needed... I really should have made a better comment, sorry!

File: AnkiDroid/src/main/java/com/ichi2/anki/CardBrowser.java [hunk 1]
```diff
@@ -1141,8 +1141,8 @@ private void closeCardBrowser(int result, Intent data) {
         public void onScroll(AbsListView view, int firstVisibleItem, int visibleItemCount, int totalItemCount) {
             // Show the progress bar if scrolling to given position requires rendering of the question / answer
             int lastVisibleItem = firstVisibleItem + visibleItemCount;
-            // Don't try to start rendering before scrolling has begun (firstVisibleItem == 0)
-            if (firstVisibleItem > 0 && lastVisibleItem > 0 && getCards().size() > lastVisibleItem) {
+            // Don't try to start rendering before scrolling has begun
+            if (lastVisibleItem > 0 && getCards().size() > lastVisibleItem) {
```
Comment (hssm) [inline AnkiDroid/src/main/java/com/ichi2/anki/CardBrowser.java]: `getCards()` returns and empty list results in an `IndexOutOfBoundsException`. It seems that we're trying to render before we get any results on the first load. I can change it to ```if (getCards().size() > 0)``` which seems to work just as well.

Comment (timrae) [inline AnkiDroid/src/main/java/com/ichi2/anki/CardBrowser.java]: Probably we should at least check explicitly that `firstVisibleItem` and `lastVisibleItem` are in bounds.

**Comment by hssm** [comment]
How's this?

**Comment by timrae** [comment]
Yeah let's see how it goes

**Pull request merged**